src/math.rs

```diff
@@ -1,11 +1,21 @@
 use vecmath::Matrix4;
 use quaternion::Quaternion;
+use std::mem;
+
+pub fn inv_sqrt(x: f32) -> f32 {
```

Comment:
I think the `inv_sqrt` hack doesn't give much performance benefit on modern hardware.